src/chainer.js

```diff
@@ -8,11 +10,13 @@ function chainer(functions) {
     let result = x;
 
     for (const f of functions) {
-      result = f(result);
+      if (typeof f === 'function') {
+        result = f(result);
+      }
     }
 
     return result;
-  }
```

Comment:
There is an extra semicolon after the function block. It is not necessary in this context.